### PR TITLE
added cors settings for ocp3

### PR DESCRIPTION
**Pull request by pranavgaikwad**

### CODE REVIEW

File: cors/cors.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    service:
+      name: "{{ item }}"
+      state: restarted
+    loop:
+      - atomic-openshift-master-api
+      - atomic-openshift-master-controllers
+    become: yes
+    when: status.rc == 0
+
+  - name: "Applying new configuration [master-restart]"
```
Comment (jwmatthews) [inline cors/cors.yaml]: Do we need both of these approaches for restarting the api and controller?

Comment (pranavgaikwad) [reply cors/cors.yaml]: @jwmatthews Old openshift 3 versions use the systemd controlled master node processes. In newer versions, both the processes run as pods. Hence, the two approaches. Either way, this playbook runs only one at a time.

Comment (jwmatthews) [reply cors/cors.yaml]: thx makes sense

**Comment by jwmatthews** [comment]
ACK

**Pull request merged**